lib/fungus_toast/games/grid.ex

```diff
@@ -53,13 +53,18 @@ defmodule FungusToast.Games.Grid do
     trunc(x_coordinate + grid_height_and_width * y_coordinate)
   end
 
+  def generate_growth_cycles(starting_grid, grid_size, player_id_to_player_map, growth_cycle, acc \\ [])
   @spec generate_growth_cycles(map(), integer(), map(), integer()) :: any()
-  def generate_growth_cycles(starting_grid, grid_size, player_id_to_player_map, number_of_growth_cycles) do
+  def generate_growth_cycles(starting_grid, grid_size, player_id_to_player_map, growth_cycle, acc) when growth_cycle > 0 do
```

Comment:
I would personally remove the guard here and just set line 66 to a 0. Much cleaner and easier to read

Reply:
Or leave this and do the 0 on line 66. If someone passed in a negative number, then it would at least error